src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingGuardTower.java

```diff
@@ -7,54 +7,55 @@
 import org.jetbrains.annotations.NotNull;
 
 /**
- * Building class of the BarracksTower.
+ * Guard Tower building.
+ *
+ * @author Asherslab
  */
+@SuppressWarnings("squid:MaximumInheritanceDepth")
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Suppressing the 'squid:MaximumInheritanceDepth' warning is not allowed [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1309)